src/pipelines/features/RunMessages/RunMessages.tsx

```diff
@@ -48,11 +48,22 @@ const RunMessages = (props: RunMessagesProps) => {
         label={t("Timestamp")}
         format={DateTime.DATETIME_SHORT_WITH_SECONDS}
         defaultValue="-"
+        className="max-w-[50ch] py-3"
       />
-      <BaseColumn accessor="priority" label={t("Priority")}>
+      <BaseColumn
+        accessor="priority"
+        label={t("Priority")}
+        className="max-w-[50ch] py-3"
+      >
         {(value) => <Badge className={getBadgeClassName(value)}>{value}</Badge>}
       </BaseColumn>
-      <TextColumn width={400} accessor="message" label={t("Message")} />
+      <BaseColumn accessor="message" label={t("Message")} width={400}>
+        {(value) => (
+          <MarkdownViewer className="w-max-[30ch] text-sm overflow-y-auto whitespace-pre-line break-words">
```

Comment:
Could we forward `allowedElements={['a']}` to the underlying component? We don't want to support other tags at this stage.

https://github.com/remarkjs/react-markdown#options

Reply:
If we do this it'll prevent all the others tag from being rendered (in our case message will be empty)

Reply:
In that case we'll also add `allowedElements={['p','a']}`